sample-apps/react-native/dogfood/src/components/MeetingUI.tsx

```diff
@@ -30,6 +32,8 @@ export const MeetingUI = ({
 }: Props) => {
   const [selectedMode, setSelectedMode] = React.useState<Mode>('grid');
   const call = useCall();
+  const channelWatched = useChannelWatch();
+  const unreadCount = useUnreadCount({ channelWatched });
```

Comment:
```suggestion
  const unreadBadgeCountIndicator = useUnreadCount({ channelWatched });
```

Reply:
This hook returns a number, so probably unreadCount made more sense